presto-native-execution/presto_cpp/main/common/Utils.cpp

```diff
@@ -55,4 +55,12 @@ long getProcessCpuTimeNs() {
   return tvNanos(rusageEnd.ru_utime) + tvNanos(rusageEnd.ru_stime);
 }
 
+void installSignalHandler() {
+#ifdef __APPLE__
+  google::InstallFailureSignalHandler();
+#else
+  facebook::velox::process::addDefaultFatalSignalHandler();
```

Comment:
Looks like the Velox header for `addDefaultFatalSignalHandler` is missing here.

Reply:
Right.